LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -176,34 +183,64 @@ class LockboxXCUITests: BaseTestCase {
         XCTAssertTrue(app.buttons["disconnectFirefoxLockbox.button"].exists, "The option to disconnect does not appear")
     }
 
-    func test3SettingOpenWebSitesIn() {
+    func test2SettingOpenWebSitesIn() {
```

Comment:
Do the tests need to be numbered?

Reply:
So far yes, the way the are written (not perfect but first approach) is to log in once and run all the tests. The number is to control the order when running the tests. 
This will be fixed in #531 where each test is independent. We have to introduce more logic and checks before each test, but at least they will be independent from previous tests and results.

Reply:
👍